guide/blueprints/custom-entities.md

```diff
@@ -46,14 +46,48 @@ before running `launch.command` relative to where that file is installed (or unp
 with the default `launch.command` being `./start.sh`.
 
 So if we create a file `/tmp/netcat-server.tgz` containing just `start.sh` in the root
-which consists of the two lines in the previous example,
+which contains the line `echo hello | nc -l 4321`, 
 we can instead write our example as: 
 
 {% highlight yaml %}
 {% readj example_yaml/vanilla-bash-netcat-file.yaml %}
 {% endhighlight %}
 
 
+#### Determining Successful Launch
+
+The default behaviour for `VanillaSoftwareProcess` is to run a command over ssh to do a 
```

Comment:
The default method used to determine a successful launch of `VanillaSoftwareProcess`